plugins/bigquery/dbt/adapters/bigquery/impl.py

```diff
@@ -9,9 +9,9 @@
 import dbt.clients.agate_helper
 
 from dbt import ui
-from dbt.adapters.base import (
-    BaseAdapter, available, RelationType, SchemaSearchMap, AdapterConfig
+from dbt.adapters.base import ( available, RelationType, SchemaSearchMap, AdapterConfig
```

Comment:
flake 8 errors are reported as `line:position`, sort of like biblical verses. So this error is on line 12, position 32 (`12:13`), and line 13, position 1 (`13:1`):
```
plugins/bigquery/dbt/adapters/bigquery/impl.py:12:32: E201 whitespace after '('
plugins/bigquery/dbt/adapters/bigquery/impl.py:13:1: E124 closing bracket does not match visual indentation
```
The issue is the asymmetry between the open + close parentheses. I think this could be made consistent as either:
```python
from dbt.adapters.base import ( 
    available, RelationType, SchemaSearchMap, AdapterConfig
)
```
Or if it all fits on one line (and <99 characters):
```python
from dbt.adapters.base import available, RelationType, SchemaSearchMap, AdapterConfig
```

Reply:
learn something new everyday! did some reading on flake8 errors and checked the make file for running the test locally to make sure this cleared before pushing.